run.sh

```diff
@@ -68,10 +80,19 @@ run_server_jdbc() {
     --spanner_use_jdbc $@
 }
 
+run_server_python() {
```

Comment:
High-level question:
I tried to run the script in mac and had at least two issues:
- `declare -A` the bash version I had didn't know the -A option
- `fuser -K` the fuser in mac didn't know the -K option
I suspect that running in windows would be even more challenging.
The script works flawlessly in linux though.

Any ideas on how to improve compatibility? Some ideas:
a. Use python for scripting
b. Keep the script (for linux users), but also keep the more low-level instructions in case the script wouldn't work in the environment

This doesn't have to be done as part of this PR, but it's a good discussion to start.